build-package.sh

```diff
@@ -658,6 +666,7 @@ for ((i=0; i<${#PACKAGE_LIST[@]}; i++)); do
 					 $(test "${TERMUX_WITHOUT_DEPVERSION_BINDING:-}" = "true" && echo "-w") \
 					 $(test -n "${TERMUX_PACKAGE_FORMAT:-}" && echo "--format $TERMUX_PACKAGE_FORMAT") \
 					 $(test -n "${TERMUX_PACKAGE_LIBRARY:-}" && echo "--library $TERMUX_PACKAGE_LIBRARY") \
+					 $(test "${TERMUX_DEPENDENCY_DOWNLOAD_PARALLELIZING-true}" = "false" ]] && echo "--disable-dependency-download-parallelizing") \
```

Comment:
```suggestion
					 $(test "${TERMUX_DEPENDENCY_DOWNLOAD_PARALLELIZING-true}" = "false" ]] && echo "--no-parallel") \
```

Not for this PR but for later.
A potential improvement could be making `$TERMUX_DEPENDENCY_DOWNLOAD_PARALLELIZING` an int value instead of a bool to set the maximum concurrency.

To that end a better name might be `$TERMUX_MAX_PARALLEL_DOWNLOADS`.

Reply:
Same as above.